src/components/Combobox.vue

```diff
@@ -99,6 +99,13 @@ export default defineComponent({
     inline: {
       type: Boolean,
       default: false
+    },
+    /**
+     * allow the options panel to be rendered outside the flow of a container that has content that needs scrolling
+     */
+    escapeOverflow: {
+      type: Boolean,
+      default: false
```

Comment:
If PR introduces a new prop, I would expect to have at least one new story for Combobox and Listbox sections? Not only for Dialog.

Reply:
done